### PR TITLE
function for getting secrets in a workflow

### PR DESCRIPTION
**Pull request by ayushkamat**

### CODE REVIEW

File: latch/functions/secrets.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import os
+
+from latch_cli.config.latch import _LatchConfig
+from latch_cli.tinyrequests import post
+from latch_cli.utils import current_workspace, retrieve_or_login
+
+config = _LatchConfig()
```
Comment (maximsmol) [inline latch/functions/secrets.py:7]: should probably be a singleton

File: latch/functions/secrets.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+endpoints = config.sdk_endpoints
```
Comment (maximsmol) [inline latch/functions/secrets.py:8]: never really understood why this is even a thing

File: latch/functions/secrets.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+
+def get_secret(secret_name: str):
```
Comment (maximsmol) [inline latch/functions/secrets.py:11]: shouldn't this be in the SDK part of latch and not in CLI?

**Review by maximsmol — CHANGES_REQUESTED** [review]
not in scope but could use CLI commands for listing and creating secrets

File: latch/functions/secrets.py (hunk 3 above)
Comment (ayushkamat) [reply latch/functions/secrets.py:11]: its not in the cli?

**Review by AidanAbd — APPROVED** [review]

**Pull request merged**